examples/core-hello-world/src/main/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%-5level %date %logger{0}:%thread %message%n</pattern>
+        </encoder>
+    </appender>
+
+    <!-- Some libraries are just too verbose -->
+    <logger name="akka" level="INFO"/>
+    <logger name="native-zk-connector" level="WARN" />
+    <logger name="org.apache.zookeeper" level="WARN" />
+
+    <root level="DEBUG">
+        <appender-ref ref="stdout"/>
+    </root>
+</configuration>
```

Comment:
I think we should add the configuration for structured logging. What do you think, @takirala?

Reply:
👍 we have one example at https://github.com/mesosphere/usi/blob/master/examples/hello-world/src/main/resources/logback.groovy

Reply:
yeah, let's go back to what @takirala did in that link, please :)